ocamldoc/odoc_html.ml

```diff
@@ -2418,7 +2419,7 @@ class html =
         let f_ele e =
           let simple_name = Name.simple (name e) in
           let father_name = Name.father (name e) in
-          if father_name = "Stdlib" && father_name <> simple_name then
+          if strip_stdlib && father_name = "Stdlib" && father_name <> simple_name then
```

Comment:
You could use `!Odoc_global.library_namespace` (which is set to `Stdlib` by the `-lib` option) rather than just `"Stdlib"`. But this is a quite theoretical distinction at this point.

Reply:
I'm not going to make this change in this PR: it would change behaviour for non-stdlib users of ocamldoc who pass the `-lib` option. Currently, the removal of module aliases from the index is specific to `"Stdlib"`.

Reply:
It is perfectly fine with me. Note that I am not sure if there are non-stdlib users of `-lib`.

Reply:
Ah, you're right. I'd thought it was widely-used, but I can't find any uses outside stdlib and it's marked "very experimental" in the help text. I no longer have qualms about changing it! Changed.